08_lisp.md

```diff
@@ -240,10 +268,11 @@ Return the next, otherwise the user will have to save it before freeing.
       return cdr;
     }
 
-This is the same as `(setf (cdr x) free-list)` in Lisp or `(set-cdr! x free-list)`
-in Scheme.
+[^free-in-lisp]
+We make it somewhat more useful by returning something other than void.
+Return the `next`, otherwise the user will have to save it before freeing.
```

Comment:
This sentence was a little hard to parse. Can I suggest this:
```suggestion
We will make it return `next`, otherwise the user will have to save it before freeing.
```
I think it could still be a bit clearer, but that was the most minimal change I could think of.